docs/pages/guides/configuring-ota-updates.md

```diff
@@ -10,11 +10,11 @@ OTA updates are controlled by the [`updates` settings in app.json](../workflow/c
 
 ## Automatic Updates
 
-By default, Expo will check for updates automatically when your app is launched and will try to fetch the latest published version. If a new bundle is available, Expo will attempt to download it before launching the experience. If there is no network connection available, or it has not finished downloading in 30 seconds, Expo will fall back to loading a cached version of your app, and continue trying to fetch the update in the background (at which point it will be saved into the cache for the next app load).
+If your `app.json` does not contain an `updates.fallbackToCacheTimeout` field, expo-updates will check for updates automatically when your app is launched and will try to fetch the latest published version. If a new bundle is available, Expo will attempt to download it before launching the experience. If there is no network connection available, or it has not finished downloading in 30 seconds, Expo will fall back to loading a cached version of your app, and continue trying to fetch the update in the background (at which point it will be saved into the cache for the next app load). We strongly advise against depending blocking app loading on fetching an update like this because it can leave users with the perception that the app is not working if it takes a while to download the update, and every launch will be slower because a network request has to be made to check for an update before proceeding to the app. Note that all apps created with `expo-cli` include the `updates.fallbackToCacheTimeout` field in `app.json` to override the timeout.
```

Comment:
```suggestion
If your `app.json` does not contain an `updates.fallbackToCacheTimeout` field, expo-updates will check for updates automatically when your app is launched and will try to fetch the latest published version. If a new bundle is available, Expo will attempt to download it before launching the experience. If there is no network connection available, or it has not finished downloading in 30 seconds, Expo will fall back to loading a cached version of your app, and continue trying to fetch the update in the background (at which point it will be saved into the cache for the next app load). We strongly advise against blocking app loading on fetching an update because it can leave users with the perception that the app is not working if it takes a while to download the update, and every launch will be slower because a network request has to be made to check for an update before proceeding to the app. Note that all apps created with `expo-cli` include the `updates.fallbackToCacheTimeout` field in `app.json` to override the timeout.
```

just changed to `...We strongly advise against blocking app loading on fetching an update because...`